ulib/axlibc/src/sys.rs

```diff
@@ -9,21 +9,19 @@ const PAGE_SIZE_4K: usize = 4096;
 #[no_mangle]
 pub unsafe extern "C" fn ax_sysconf(name: c_int) -> c_long {
```

Comment:
Add `sys_sysinfo` in `arceos_posix_api`, and we can remove the `axalloc` dependency.